x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/resource_tab.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiAccordion,
+  EuiCode,
+  EuiCodeBlock,
+  EuiDescriptionList,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { getFlattenedObject } from '@kbn/std';
+import { CspFinding } from '../types';
+import * as TEXT from '../translations';
+
+const getDescriptionDisplay = (value: any) => {
+  if (typeof value === 'boolean' || value === undefined || value === null) {
+    return <EuiCode>{JSON.stringify(value)}</EuiCode>;
+  }
+
+  if (typeof value === 'object') {
+    return (
+      <EuiCodeBlock isCopyable={true} overflowHeight={300}>
+        {JSON.stringify(value, null, 2)}
+      </EuiCodeBlock>
+    );
+  }
+
+  return <EuiText size="s">{value}</EuiText>;
+};
+
+const prepareDescriptionList = (data: Record<string, any>) =>
```

Comment:
Please don't use `any`, use `unknown` instead if this method should not be aware of the value types